tcf_website/tests/test_commands.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for Django management commands"""
+from django.core import management
+from django.test import TestCase
+
+from tcf_website.models import CourseGrade, CourseInstructorGrade
+from .test_utils import setup
+
+
+class LoadGradesTestCase(TestCase):
+    """ Tests for the load_grades command. Uses the test_data.csv file, which holds dummy
+    data for two sections of the same course taught by the same instructor."""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        setup(cls)
+        management.call_command('load_grades', 'test_data', '--suppress-tqdm', verbosity=0)
+        cls.cg = CourseGrade.objects.all()[0]
+        cls.cig = CourseInstructorGrade.objects.all()[0]
+
+    def test_no_duplicates(self):
+        """Make sure only one instance of CourseGrade and CourseInstructorGrade were created"""
+        self.assertEqual(len(CourseGrade.objects.all()), 1)
+        self.assertEqual(len(CourseInstructorGrade.objects.all()), 1)
```

Comment:
```suggestion
        self.assertEqual(CourseGrade.objects.count(), 1)
        self.assertEqual(CourseInstructorGrade.objects.count(), 1)
```

Reply:
Technically the CSV data file is CS 420, but I'll just update the CSV to reflect this because we want 4-digit class codes.